### PR TITLE
Docker Image for AWS Tutorial

### PR DESCRIPTION
**Pull request by bmhan12**
This PR:
- Adds a Docker image for the AWS Tutorial
- Image builds with CUDA, ~assumes V100 availability (TBD)~ targetting Tesla  T4 GPUs

- [x] Test that image builds locally through Docker Desktop
- [x] Test that VSCode works on port 3000 when image is run locally
- [x] Build Docker image on AWS AMI
- [x] Test VSCode works when image is run on AMI
- [x] Verify can run with CUDA hardware on AWS
- [x] Create AWS AMI running VSCode with CUDA hardware
- [x] Iterate on, create final AMI version for tutorial

### CODE REVIEW

**Comment by bmhan12** [comment]
I wrote down a working set of notes for setting up the AWS instance on Confluence: https://lc.llnl.gov/confluence/display/ASCT/AWS+Setup+Working+Notes

**Review by rhornung67 — APPROVED** [review]
Thanks @bmhan12 for working through this.

File: scripts/docker/dockerfile_gcc-9_cuda-11 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# Tutorial branch
+ARG branch=feature/han12/docker_tutorial
+ARG USER=axomdev
+ENV HOME /home/${USER}
+
+SHELL ["/bin/bash", "-c"]
+RUN sudo apt-get update -y
+RUN sudo apt-get install -y supervisor
+RUN sudo useradd --create-home --shell /bin/bash ${USER}
+RUN sudo apt-get install doxygen gfortran graphviz language-pack-en-base less libopenblas-dev libomp-dev mpich python3-sphinx ssh texlive-full tree -fy
```
Comment (kennyweiss) [inline scripts/docker/dockerfile_gcc-9_cuda-11:17]: 👍 -- thanks for adding these!

**Review by kennyweiss — APPROVED** [review]
Thanks @bmhan12 !

File: scripts/docker/supervisord.conf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[supervisord]
```
Comment (white238) [inline scripts/docker/supervisord.conf:1]: Add a comment for what this file is for then merge imo.

**Pull request merged**